tests/test_happy_hours.py

```diff
@@ -32,52 +32,53 @@ def test_happy_hour():
 
     assert hh_block == HappyHourBlock(
         type="happy_hours",
-        status="yes",
+        status="open",
         next_transition_datetime="2018-06-14T22:00:00+03:00",
         seconds_before_next_transition=1800,
+        is_24_7=False,
         raw="Mo-Sa 20:00-22:00; Su 12:00-14:00, 20:00-22:00",
         days=[
             {
                 "dayofweek": 1,
                 "local_date": "2018-06-11",
-                "status": "yes",
-                "happy_hours": [{"beginning": "20:00", "end": "22:00"}],
+                "status": "open",
+                "opening_hours": [{"beginning": "20:00", "end": "22:00"}],
```

Comment:
As discussed, I am okay with a breaking change as this block is not currently used.

But does it really make sense to talk about happy hours as "open" and "closed" ?

Reply:
Well, that's a good point, I suppose I got lost in the abstractions... ^^

Reply:
If that helps with the maintenance, we could simply delete this block for now, and we'll agree on its format when its use case will be clearer.

Reply:
I went for this solution. It seems quite lazy but I suppose it wouldn't be a good idea to keep maintaining unused stuff by default either :|